### ISSUE TITLE
Nummular eczema

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6077]
**Preferred term label:**
Nummular eczema

**Synonyms**
Nummular dermatitis, Discoid eczema

**Definition (free text, please give PubMed ID)**
An allergic disorder with pruritic coin-shaped patches on the skin
PMID: 26091664

**Parent term (use hpo.jax.org/app)**
Atopic eczema

**Comment by pnrobinson** [comment]
added as HP:0033120
